client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -147,6 +147,8 @@ const Task = ( {
 							className="task__skip is-link"
 							ref={ skipButtonRef }
 							onClick={ () => ( enableSkipOptions ? setSkipOptionsVisible( true ) : skipTask() ) }
+							aria-haspopup
+							aria-expanded={ enableSkipOptions && areSkipOptionsVisible }
```

Comment:
The [WAI recommendation](https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-14) is to not include the `aria-expanded` attribute rather than leave it false.

```suggestion
							aria-expanded={ enableSkipOptions && areSkipOptionsVisible || undefined }
```

Reply:
Another thing that can be added is the `aria-controls` attribute. So you'd add something like `aria-controls="unique-id-123"` to this button, and then add `id="unique-id-123"` to the `<PopoverMenu>` component.

One way to get a unique id for your components is to use the `withInstanceId` HOC
https://github.com/automattic/wp-calypso/blob/0fee1ca86a95b29d58ea3c207ffd13d983eadb44/packages/page-pattern-modal/src/components/page-pattern-modal.tsx#L10

Reply:
Cool! I also want to add `aria-controls` but I'm not familiar to get a unique id. It's very useful 👍